.github/workflows/pull-request-comments.yml

```
@@ -163,3 +163,46 @@ jobs:
             `;
 
             github.rest.issues.createComment( commentInfo );
+
+  # Leaves a comment on a pull request when no Trac ticket is included in the pull request description.
+  trac-ticket-check:
+    name: Comment on a pull request when no Trac ticket is included
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    if: ${{ github.repository == 'WordPress/wordpress-develop' && github.event_name == 'pull_request_target' }}
```

```suggestion
    if: ${{ github.repository == 'WordPress/wordpress-develop' && github.event_name == 'pull_request_target' && github.event.pull_request.draft == false }}
```

It would be good to wait until a PR is out of draft before adding this comment. 

When starting on a ticket, it's sometimes helpful to leave out the link to the ticket intentionally so the PR doesn't show up on trac until the code is ready to be reviewed.

---

Good call, @peterwilsoncc! There's some tested logic around this in the Props Bot workflow we could borrow here.